cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -242,48 +242,51 @@ func runImport(ctx context.Context, varMap map[string]string, importWorkflowPath
 // Run runs import workflow.
 func Run(clientID string, imageName string, dataDisk bool, osID string, customTranWorkflow string,
 	sourceFile string, sourceImage string, noGuestEnvironment bool, family string, description string,
-	network string, subnet string, zone string, timeout string, project *string,
+	network string, subnet string, zone string, timeout string, project string,
 	scratchBucketGcsPath string, oauth string, ce string, gcsLogsDisabled bool, cloudLogsDisabled bool,
 	stdoutLogsDisabled bool, kmsKey string, kmsKeyring string, kmsLocation string, kmsProject string,
 	noExternalIP bool, labels string, currentExecutablePath string, storageLocation string,
-	uefiCompatible bool) (*daisy.Workflow, error) {
+	uefiCompatible bool) (*daisy.Workflow, map[string]string, error) {
+
+	updatedParams := map[string]string{}
 
 	log.SetPrefix(logPrefix + " ")
 
 	sourceBucketName, sourceObjectName, userLabels, err := validateAndParseFlags(clientID, imageName,
 		sourceFile, sourceImage, dataDisk, osID, customTranWorkflow, labels)
 	if err != nil {
-		return nil, err
+		return nil, updatedParams, err
 	}
 
 	ctx := context.Background()
 	metadataGCE := &compute.MetadataGCE{}
 	storageClient, err := storage.NewStorageClient(
 		ctx, logging.NewLogger(logPrefix), oauth)
 	if err != nil {
-		return nil, err
+		return nil, updatedParams, err
 	}
 	defer storageClient.Close()
 
 	scratchBucketCreator := storage.NewScratchBucketCreator(ctx, storageClient)
 	computeClient, err := param.CreateComputeClient(&ctx, oauth, ce)
 	if err != nil {
-		return nil, err
+		return nil, updatedParams, err
 	}
 	zoneRetriever := storage.NewZoneRetriever(metadataGCE, computeClient)
 
 	region := new(string)
-	err = param.PopulateMissingParameters(project, &zone, region, &scratchBucketGcsPath,
+	err = param.PopulateMissingParameters(&project, &zone, region, &scratchBucketGcsPath,
 		sourceFile, metadataGCE, scratchBucketCreator, zoneRetriever, storageClient)
+	updatedParams["project"] = project
```

Comment:
Extract "project" as a constant to a common place shared across all cli tools. Maybe in service package?